### ISSUE TITLE
Can convert temperature to non-temperature units

### ISSUE THREAD

**Issue by jrleeman** [hgrecco/pint#523]
Temperature can be converted to just about anything, including seconds, meters, teslas, etc. Example:

```python
import pint
units = pint.UnitRegistry()

x = 10 * units.km
print(x.to('degF'))

x = 10 * units.seconds
print(x.to('degF'))

x = 10 * units.tesla
print(x.to('degF'))
```

Produces
```
17540.330000399998 degF
-441.6699996 degF
17540.330000399998 degF
```

**Comment by dalito** [comment]
Interesting that this bug has not been discovered before. I have a fix for this and can submit a PR tomorrow.

**Comment by hgrecco** [comment]
This is fixed in 0.8.1. Awesome fast response!

**Comment by jrleeman** [comment]
Awesome fast indeed! Thanks @dalito et al!
